### PR TITLE
add info about product files

### PR DESCRIPTION
**Pull request by TomasPilar**
@Machy8 mrkni zda je to srozumitelné, díky!

### CODE REVIEW

File: docs/api/store-api.md [hunk 1]
```diff
@@ -6,6 +6,25 @@ Apicart Store API listen at [https://store-api.apicart.net](https://store-api.ap
 Full GraphQL schema for Store API is available [here](https://docs.store-api.apicart.net).
 :::
 
+
+## Item files
+
+Mutation handling items in cart (e.g. `addCartItems` or `removeCartItems`) getting data from item files. These files describe the item and define, for example, its name or price.
```
Comment (Machy8) [inline docs/api/store-api.md:12]: Mutations manipulating with products .... are getting data from product files. These files describes... and defines for example its name or price.

File: docs/api/store-api.md [hunk 2]
```diff
@@ -6,6 +6,25 @@ Apicart Store API listen at [https://store-api.apicart.net](https://store-api.ap
...
+
+These files ensure the security and validity of the data sent to the API. A potential attacker cannot add a item to a cart at a price other than that listed in the item file.
```
Comment (Machy8) [inline docs/api/store-api.md:14]: These files ensures that product data sent to our API are valid and makes shure that for example a product price cannot be modified by some attacker.

File: docs/api/store-api.md [hunk 3]
```diff
@@ -6,6 +6,25 @@ Apicart Store API listen at [https://store-api.apicart.net](https://store-api.ap
...
+
+Item files are created automatically through the "Item" module in Apicart administration and their URLs are always on the item detail. You can also create these files yourself if you have the option to post them to some publicly available URL.
```
Comment (Machy8) [inline docs/api/store-api.md:16]: Product files... "Item module"... in the Apicart administration... on the item detail page...

File: docs/api/store-api.md [hunk 4]
```diff
@@ -6,6 +6,25 @@ Apicart Store API listen at [https://store-api.apicart.net](https://store-api.ap
...
+
+Currently supported foramt is JSON. You can view content of [valid item file here](https://sandbox.apicart.net/data/product/0001.json) (this file is actually used on https://sandbox.apicart.net/).
```
Comment (Machy8) [inline docs/api/store-api.md:18]: format... You can view a content example...

File: docs/api/store-api.md [hunk 5]
```diff
@@ -6,6 +6,25 @@ Apicart Store API listen at [https://store-api.apicart.net](https://store-api.ap
...
+
+In order to add a item to the cart it must contain four mandatory information:
```
Comment (Machy8) [inline docs/api/store-api.md:20]: ...an item

**Review by Machy8 — COMMENTED** [review]
cr done

**Comment by TomasPilar** [comment]
:+1:

**Pull request merged**